test/controllers/events_controller_test.rb

```diff
@@ -147,6 +161,27 @@ def setup_live_events
       assert_match("data-name='w'  selected", @response.body)
```

Comment:
Shouldn't it select the day aggregation if we are at the `7d` date range?

Reply:
I don't understand what you mean 😅 this is the test for the 30d aggregation, right?